### PR TITLE
Add variable to playwright tests to use stage apis

### PR DESCRIPTION
**Pull request by kenoir**
## Who is this for?

Folk who want to run the playwright tests with stage APIs

## What is it doing for them?

Allowing the catalogue-api smoke tests to ensure the current prod deploy of the wellcomecollection.org site will work with what's in stage - thereby indicating it is safe to deploy catalogue-api stage to prod.

### CODE REVIEW

File: .buildkite/pipeline.e2e.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+steps:
+  - label: "e2e test:desktop"
+    plugins:
+      - docker-compose#v3.5.0:
+          run: e2e
+          env:
+            - PLAYWRIGHT_BASE_URL
+            - USE_STAGE_APIS
```
Comment (jamesgorrie) [inline .buildkite/pipeline.e2e.yml:8]: Where does this get set?

File: playwright/test/helpers/urls.ts [hunk 2]
```diff
@@ -3,3 +3,4 @@ export const baseUrl = process.env.PLAYWRIGHT_BASE_URL
   : 'http://localhost:3000';
 export const worksUrl = `${baseUrl}/works`;
 export const imagesUrl = `${baseUrl}/images`;
+export const useStageApis = 'USE_STAGE_APIS' in process.env;
```
Comment (jamesgorrie) [inline playwright/test/helpers/urls.ts:6]: We don't care about the value, just that it exists? e.g. `USE_STAGE_APIS=false`?

**Review by jamesgorrie — APPROVED** [review]

File: .buildkite/pipeline.e2e.yml (hunk 1 above)
Comment (kenoir) [reply .buildkite/pipeline.e2e.yml:8]: This specifies that these env vars are settable - when you trigger the build from another pipeline you add the env vars to that. https://buildkite.com/docs/pipelines/trigger-step#trigger-step-attributes

Comment (jamesgorrie) [reply .buildkite/pipeline.e2e.yml:8]: 👍

File: playwright/test/helpers/urls.ts (hunk 2 above)
Comment (kenoir) [reply playwright/test/helpers/urls.ts:6]: Yeah, but actually I think I prefer checking for a value - will fix.

**Pull request merged**